client/src/modules/patients/edit/edit.html

```diff
@@ -108,6 +108,18 @@
                   </div>
                 </div>
 
+                 <!-- Reference Name -->
+                 <div
+                 class="form-group has-feedback"
+                 ng-class="{'has-error' : patientDetails.display_name.$invalid && patientDetails.$submitted}">
+
+                 <label class="col-md-3 control-label" translate> FORM.LABELS.REFERENCE </label>
+                 <div class="col-md-9">
+                   <input class="form-control" name="reference" disabled="disabled" ng-model="PatientEditCtrl.medical.reference">
```

Comment:
Instead of a disabled input (which seems like the user should be able to edit at some point), bootstrap gives us another class called `form-control-static` if the input will never be editable.  We used it right above this input [in the registration date section](https://github.com/IMA-WorldHealth/bhima/pull/3545/files#diff-d1558a6f91b42148dcbbe4563ad75eaeR107).

Could you change this to `<p class="form-control-static">` and put the reference inside it?